src/utils/IverifiedCredential.ts

```diff
@@ -0,0 +1,58 @@
+export interface CredentialSubject {
```

Comment:
This file needs renaming to match  Pascal Case for interface filenames

Reply:
Done.